### PR TITLE
Added FirstTimePage, a guide for first time hackers.

### PR DESCRIPTION
**Pull request by NicholasLYang**

### CODE REVIEW

File: src/js/modules/core/components/FirstTimePage.tsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  return (
+    <div className={classes.FirstTimePage}>
+      <h1> First Time Hacker Guide </h1>
+      <div className={classes.content}>
+        <p>
+          First time at a hackathon? Feeling overwhelmed? Not to worry! We've
+          prepared a guide for first time hackers (or anybody looking for
+          awesome activities).
+        </p>
+        <h2 className={classes.sectionHeader}> Bucket List </h2>
```
Comment (themichaelyang) [inline src/js/modules/core/components/FirstTimePage.tsx]: big fan of this

File: src/js/modules/core/components/FirstTimePage.tsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        <ul>
+          <li> Attend a workshop! </li>
+          <li> Win a cup stacking competition! </li>
+          <li> Go to team hunt! </li>
+          <li> Chat with a sponsor! </li>
+          <li> Borrow from the MLH Hardware Lab! </li>
+          <li> Decorate some cupcakes! </li>
+          <li> Play pool at midnight </li>
+        </ul>
+        <h2 className={classes.sectionHeader}> Ideas &amp; Projects </h2>
```
Comment (themichaelyang) [inline src/js/modules/core/components/FirstTimePage.tsx]: I wonder if we can have links to fun project ideas or learning resources?

Comment (NicholasLYang) [reply src/js/modules/core/components/FirstTimePage.tsx]: Got any good ones? I was considering linking a Paul Graham essay but it might be too weighty

Comment (themichaelyang) [reply src/js/modules/core/components/FirstTimePage.tsx]: For total beginners who want a taste:
https://www.codecademy.com/

For total beginners that are Very Serious:
https://ocw.mit.edu/courses/electrical-engineering-and-computer-science/6-0001-introduction-to-computer-science-and-programming-in-python-fall-2016/

Comment (NicholasLYang) [reply src/js/modules/core/components/FirstTimePage.tsx]: Ehh...I'm not sure codecademy is the best use of someone's time at a hackathon. Realistically they'll get through like 2% before getting bored.

Comment (NicholasLYang) [reply src/js/modules/core/components/FirstTimePage.tsx]: If someone literally knows *zero* programming, they'll need more help than Codecademy

Comment (themichaelyang) [reply src/js/modules/core/components/FirstTimePage.tsx]: i dunno if they should be at the hackathon then tbh, but our faq was clear about what we meant as a beginner. i think its better than nothing to put a link to an tutorial + zero setup (afterwards mentors can help them set up an environment)

Comment (NicholasLYang) [reply src/js/modules/core/components/FirstTimePage.tsx]: Yeah I just think that codecademy won't be the best activity at a hackathon. Ideally beginners should have more guidance than that.

**Pull request merged**